lms/templates/ccx/schedule.html

```diff
@@ -29,80 +29,88 @@
   <div id="new-ccx-schedule"></div>
 </div>
 
-<section id="enter-date-modal" class="modal" aria-hidden="true">
-  <div class="inner-wrapper" role="dialog">
-    <button class="close-modal">
-      <i class="icon fa fa-remove"></i>
+<section id="enter-date-modal" class="modal"
+         tabindex="1" role="dialog" aria-labelledby="ccx_schedule_set_date_heading">
```

Comment:
a positive integer should never be used as a value for ```tabindex```.  Use ```-1``` if you intend to use script to move focus to an object (and never allow the user to move focus to the element directly) and use ```0``` if you want to force it into the natural tab order.  In this instance, you are moving focus with script, so make it a ```-1```